src/components/store/features/notifications/notification-slice.ts

```diff
@@ -15,24 +22,71 @@ const notificationAdapter = createEntityAdapter({
   selectId: (notification: NotificationItem) => notification.id
 });
 
+interface NotificationState
+  extends ReturnType<typeof notificationAdapter.getInitialState> {
+  motorHot: boolean;
+}
+
+const initialState: NotificationState = {
+  ...notificationAdapter.getInitialState(),
+  motorHot: false
+};
+
+export const cleanMessage = (message: string, key: string | null): string =>
+  key ? message.replace(key, '').trim() : message;
+
+export const processNotification = (
```

Comment:
I dont really like this, can we not add it to the state message?
Have the notification inform them and then do the rest in the backend where we prevent the starting?

Reply:
I feel like the machine status should not be part of the notification logic and we already have machine states